src/Microsoft.ML.Transforms/Expression/TokKind.cs

```diff
@@ -95,37 +95,6 @@ internal enum TokKind
         And,
         Or,
 
-        // REVIEW: These are specific to the NetParser. Use a general mechanism.
-        Const,
-        Input,
-        Output,
-        Hidden,
-        Share,
-        Sigmoid,
-        Linear,
-        SoftMax,
-        RectifiedLinear,
-        Square,
-        Sqrt,
-        SoftRectifiedLinear,
-        Tanh,
-        BoundedRectifiedLinear,
-        From,
-        All,
-        Where,
-        Convolve,
-        Pool,
-        Abs,
-        Bittest,
-        Max,
-        Mean,
-        Response,
-        Norm,
-        FloatsFromBytes,
-        Param,
-        Auto,
```

Comment:
Why these are removed?

Reply:
See: https://github.com/dotnet/machinelearning/pull/4548#discussion_r360970284
These are used for the internal NN language, Net#. If we are no longer supporting that internally, then they are no longer needed. I don't think this PR provides the code for those functions, like convolution.

We may want to check that `abs(a)`, `min(a, b)`, `max(a,b )`, `sqrt(a)` still work.